java/ql/src/utils/GenerateFlowTestCase.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python3
+
+import errno
+import json
+import os
+import os.path
+import re
+import shlex
+import shutil
+import subprocess
+import sys
+import tempfile
+
+if any(s == "--help" for s in sys.argv):
+  print("""Usage:
+GenerateFlowTestCase.py specsToTest.ssv projectPom.xml outdir
```

Comment:
I'd prefer to s/ssv/csv/ throughout whenever we need the abbreviated form (as for example here in a file extension), and then either using csv or semicolon-separated-values elsewhere (I consider CSV to be the catch-all generic abbreviation for delimiter-separated-values).